liberty-maven-plugin/src/it/appsdirectory-notset-configured-it/src/test/java/net/wasdev/wlp/maven/test/app/DefaultAppDirectoryTest.java

```diff
@@ -79,4 +79,22 @@ public void testApplicationFileExist() throws Exception {
         File f = new File("liberty/usr/servers/test/apps/appsdirectory-notset-configured-it.war");
         Assert.assertTrue(f.getCanonicalFile() + " doesn't exist", f.exists());
     }
+
+    @Test
+    public void testCopyDependenciesFilesExist() throws Exception {
+        File f = new File("liberty/usr/servers/test/lib/global/commons-logging-1.0.4.jar");
```

Comment:
That path looks unusual.. isn't it usually:  `target/liberty/wlp/usr/servers...`  ?

Reply:
It's relative to the particular test directory. I copied the path from the existing test method testApplicationFileExist() and I assure you it works. If I change the filename or path, it fails.

Reply:
OK, I think it's a different default when you use an           `<assemblyArtifact>` as the install source in config.